media/js/firefox/os/firefox-os-new.js

```diff
@@ -170,7 +157,14 @@
             $provider.show();
 
             // setup GA event tracking on telecom provider exit links
-            $('#provider-links a').on('click', trackProviderExit);
+            $('#provider-links a').each(function() {
+                var $this = $(this);
+
+                $this.attr({
```

Comment:
Is this kind of thing even going to be updated in the DOM before the browser navigates to the next page?